config/database.yml

```
@@ -20,6 +20,9 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: <%= ENV.fetch("DATABASE_HOST") { "localhost" } %>
+  username: <%= ENV.fetch("DATABASE_USERNAME") { %x{whoami}.chomp } %>
```

If you don't provide username it will log in as current user AFAIR
```suggestion
  username: <%= ENV["DATABASE_USERNAME"] %>
```